examples/datasets/kernel_phantom.py

```diff
@@ -0,0 +1,103 @@
+"""
+.. _ex-kernel-opm-phantom:
+
+Kernel OPM phantom data
+=======================
+
+In this dataset, a Neuromag phantom was placed inside the Kernel OPM helmet and
+stimulated with a 7 modules active (121 channels). Here we show some example traces.
+"""
+
+import numpy as np
+
+import mne
+
+data_path = mne.datasets.phantom_kernel.data_path()
+fname = mne.datasets.phantom_kernel.data_path() / "phantom_32_100nam_raw.fif"
+raw = mne.io.read_raw_fif(fname).load_data()
+events = mne.find_events(raw, stim_channel="STI101")
+
+# Bads identified by inspecting averages
+raw.info["bads"] = [
+    "RC2.bx.ave",
+    "LC3.bx.ave",
+    "RC2.by.7",
+    "RC2.bz.7",
+    "RC2.bx.4",
+    "RC2.by.4",
+    "LC3.bx.5",
+]
+# Drop the module-average channels
+raw.drop_channels([ch_name for ch_name in raw.ch_names if ".ave" in ch_name])
+# Add field correction projectors
+raw.add_proj(mne.preprocessing.compute_proj_hfc(raw.info, order=2))
+raw.pick_types(meg=True, eeg=False, stim=False, eog=False, exclude="bads")
```

Comment:
does this not work here?
```suggestion
raw.pick("meg", exclude="bads")
```